.github/workflows/release.yml

```
@@ -39,9 +44,66 @@ jobs:
           python -m twine upload --username "${TWINE_USER}" --password "${TWINE_PASSWORD}" --skip-existing --repository-url ${REPO_URL} dist/*.whl
         env:
           REPO_URL: "https://upload.pypi.org/legacy/"
-      - if: always()
-        uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
+
+  publish-lambda-layers:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: hashicorp/vault-action@v2.4.2
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/observability-team/ci/service-account/apm-agent-python access_key_id | AWS_ACCESS_KEY_ID ;
```

We could use an env variable to avoid duplication.

---

Can you elaborate?

---

```suggestion
            ${{ env.SECRET_PATH }} access_key_id | AWS_ACCESS_KEY_ID ;
```
We could define an env variable on this step and expand it in the secret input.
**Note:** It's a small cosmetic

---

9e2fe41a